react/features/base/lib-jitsi-meet/functions.any.ts

```diff
@@ -97,8 +97,7 @@ export function isFatalJitsiConnectionError(error: Error | string | ConnectionFa
     }
 
     return (
-        error === JitsiConnectionErrors.CONFERENCE_REQUEST_FAILED
-            || error === JitsiConnectionErrors.CONNECTION_DROPPED_ERROR
+        error === JitsiConnectionErrors.CONNECTION_DROPPED_ERROR
```

Comment:
Does mobile need an update too?

Reply:
This is something I have added recently and we decided against the reload, so I'm dropping that recent change.

Reply:
Go ahead and merge!